spec/commands/get_user_review_request_metrics_spec.rb

```diff
@@ -0,0 +1,105 @@
+require 'rails_helper'
+
+describe GetUserReviewRequestMetrics do
+  let(:github_user) { create(:github_user) }
+  let!(:monkeyci) { create(:github_user, login: 'monkeyci') }
+
+  def perform(*_args)
+    described_class.for(*_args)
+  end
+
+  context "without pull request" do
+    it "return empty content" do
+      expect(perform(github_user: github_user, limit_month: 9)).to eq({ "personal_mean" => 0 })
+    end
+  end
+
+  context "with pull request and review request" do
+    let(:pull_request1) do
+      create(
+        :pull_request_with_merge,
+        owner: github_user,
+        gh_created_at: DateTime.now - 5.minutes
+      )
+    end
+    let!(:review_request1) do
+      create(
+        :pull_request_review_request,
+        pull_request: pull_request1,
+        created_at: DateTime.now - 3.minutes
```

Comment:
Do not use DateTime.now without zone. Use one of Time.zone.now, DateTime.current, DateTime.now.in_time_zone, DateTime.now.utc, DateTime.now.getlocal, DateTime.now.xmlschema, DateTime.now.iso8601, DateTime.now.jisx0301, DateTime.now.rfc3339, DateTime.now.httpdate, DateTime.now.to_i, DateTime.now.to_f instead.